packages/api/src/auth/auth.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+
+import { User } from '@coderscamp/shared/models/user';
+
+import { UsersService } from '../users/users.service';
+import { GithubUserData } from './auth.model';
+
+@Injectable()
+export class AuthService {
+  constructor(private usersService: UsersService, private jwtService: JwtService) {}
+
+  login(user: User) {
+    const { id, ...profile } = user;
+    return this.jwtService.sign({ profile, sub: id });
```

Comment:
Why did you decide to store the entire user profile inside JWT? I consider if it wouldn't be enough if we store only the user's id.